corehq/apps/registry/templates/registry/registry_list.html

```diff
@@ -0,0 +1,169 @@
+{% extends "hqwebapp/base_section.html" %}
+{% load compress %}
+{% load hq_shared_tags %}
+{% load static %}
+{% load i18n %}
+
+{% block stylesheets %}{{ block.super }}
+  {% compress css %}
+  <link type="text/less"
+        rel="stylesheet"
+        media="all"
+        href="{% static 'registry/less/registry.less' %}" />
+  {% endcompress %}
+{% endblock %}
+
+{% requirejs_main 'registry/js/registry_list' %}
+
+{% block page_content %}
+  {% initial_page_data "owned_registries" owned_registries %}
+  {% initial_page_data "invited_registries" invited_registries %}
+  <div id="data-registry-list">
+    <div class="row">
+      <div class="col-sm-8 col-md-6">
+        <h3>Data Registries</h3>
+      </div>
+      <div class="col-sm-4 col-md-3">
+        <div class="pull-right" style="margin-top: 17px; margin-bottom: 8.5px">
```

Comment:
Nitpick: I'm guessing the inline styles are to keep the button vertically aligned with the header - would it work if you put this div/button inside of the h3 instead of using the row & cols?

Reply:
I think I tried that but I'll give it another go